### DISCUSSION TITLE
一些建议

### DISCUSSION THREAD

**Discussion by huangfengye** [Baiyuetribe/kamiFaka#43] [新功能提交]
1，增加订单查询密码， 单独联系方式查询很容易泄露卡密。。。
2，增加多个发信邮箱,，随机选择一个邮箱发信。

**Comment by Baiyuetribe** [comment]
回复1：否定，这个问题开发的时候，就已经讨论过，现有的联系方式仅限2小时内有效
2：这个需求一般非常少，加入也很容易，得看开发组投票结果

**Reply by huangfengye** [reply]
联系方式查询 是否绑定cookie的？非下单电脑无法查询？？主要是买卡的人都是在一个QQ群里面的，很多人都喜欢设置QQ邮箱来收取充值卡。人家一看新人进群，就有人跑去偷卡密。。。

**Reply by Baiyuetribe** [reply]
抱歉，本系统设计之初不考虑这种情况，现已无法加入你说的需求

**Reply by Baiyuetribe** [reply]
唯一能做的是，后台添加一个开关按钮，可以手动启用或关闭联系方式查询功能

**Reply by huangfengye** [reply]
好吧。。。可以选择关闭那也是OK的。
